tools/sdk-testgen/packages/autorest.testmodeler/src/core/model.ts

```diff
@@ -239,6 +239,17 @@ export class ExampleModel {
     }
 }
 
+export class OutputVariableModel {
+    languageName: string;
```

Comment:
Is this property used in testmodeler?  I tend not to declare it in testmodeler if it's only used in language test generator.